packages/usa-input-mask/src/usa-input-mask.twig

```diff
@@ -9,9 +9,18 @@
       placeholder="{{ placeholder }}" 
       pattern="{{ pattern }}" 
       class="usa-input usa-masked" 
-      aria-describedby="{{ id }}Hint"
+      aria-describedby="{{ id }}Error"
       {% if datacharset %}data-charset="{{ datacharset }}"{% endif %}
       {% if disabled_state == 'disabled' %} disabled {%- endif %}
       {% if disabled_state == 'aria-disabled' %} aria-disabled="true" {%- endif %}
+      {% if dataErrorMessage %}data-errormessage="{{ dataErrorMessage }}"{% endif %}
+      {% if dataErrorAlphabetical %}data-errormessage-alpha="{{ dataErrorAlphabetical }}"{% endif %}
+      {% if dataErrorNumerical %}data-errormessage-num="{{ dataErrorNumerical }}"{% endif %}
```

Comment:
There was some discussion related to the correct name for this in File input PR #6168 [^1].

The **TL;DR** is that there's an established naming convention of using something like `data-errorMessage`. While not a fan of it because it seems unconventional to _other_ USWDS settings, maybe we should use for consistency in error messages.

[^1]: https://github.com/uswds/uswds/pull/6168#discussion_r1842880357

Reply:
Fixed in [c4bfd9b](https://github.com/uswds/uswds/pull/6203/commits/c4bfd9b856985b0e7a45a3ba7e2da0639350fe41)